packages/data/README.md

```diff
@@ -14,6 +14,10 @@ npm install @wordpress/data --save
 
 _This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as lower versions of IE then using [core-js](https://github.com/zloirock/core-js) or [@babel/polyfill](https://babeljs.io/docs/en/next/babel-polyfill) will add support for these methods. Learn more about it in [Babel docs](https://babeljs.io/docs/en/next/caveats)._
 
+## Tutorial
```

Comment:
I'm fine with including the section the link to the tutorial from @nerrad. However, it should be further in the document. We also could move the most important bits to this document to make the default documentation better. An alternative could be moving the tutorial to the `docs` folder.

Reply:
> An alternative could be moving the tutorial to the docs folder.

This seems like the best thing to do. If @nerrad is okay with this, I can do the work to move the tutorial series to `/docs`, perhaps as a new How-to "Getting started with Wordpress Data" guide?

Reply:
**I'm completely okay with however the content ends up getting used for the project.** However, there's some things to keep in mind when considering moving the entire tutorial:

- The series currently references examples in codesandbox.io for following along (or learning as you go). Might have to consider whether referencing those examples in the tutorial are okay. I'm not sure how easy it would be to update them if needed to keep up with any changes in the `wp.data` API.
- There's a _lot_ of content in the series (which is why it was broken apart in to parts). That's quite a bit of extra content to dump into the docs for ongoing maintenance. On the flip side, it increases the likelihood it _would_ get updated.
- The series already needs some updating. While there's nothing in it that is incorrect, there have been some recentish changes to the `@wordpress/data-controls` moving certain controls right into the `@wordpress/data` package and introducing new controls. I haven't got around to updating the series yet but if this gets moved into the official docs, it probably should be updated.
- I kind of like leaving the door open for alternative approaches to teaching this subject and I'm wary that having an official tutorial embedded in the docs might discourage that. Along with just leaving a link instead, we could also leave a little note about welcoming other tutorial link suggestions via PR for consideration? The downsides of linking out is having to make sure that link remains accurate and accessible, and the subjective weighing of whether a link should be in the docs (avoiding appearance of favoritism and all that - or maybe I worry too much there).
- I also like the idea of just selectively moving useful additional bits from the tutorial to the docs to help expand details on using the interface.